### PR TITLE
Dev1

### PR DESCRIPTION
**Pull request by akolesnikova**
Task for Phase 1

### CODE REVIEW

**Comment by akolesnikova** [comment]
@andynati Please, have a look to .circleci/config.yml and Tests/fibonacci_tests.py

File: Tests/.pytest_cache/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```
Comment (andynati) [inline Tests/.pytest_cache/README.md]: .pytest_cache directory need to be excluded from git repo. (add it to .gitignore)

**Comment by akolesnikova** [comment]
@andynati  Please, review.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6.6-stretch-browsers
+    working_directory: ~/AQA_Python
```
Comment (andynati) [inline .circleci/config.yml]: Update to your repo name -AQA-Python (replace "_" with "-")

**Comment by akolesnikova** [comment]
@andynati  Please, review

**Review by andynati — APPROVED** [review]

**Pull request merged**